app/controllers/main_controller.rb

```diff
@@ -3,17 +3,20 @@
 # controller for user model
 class MainController < ApplicationController
   def index
-    @users = User.all
-
-    @users.each do |user|
-      unless user.avatar.attached?
-        user.avatar.attach(io: File.open("storage/user-images/user-default.jpeg"),
-                           filename: "file.pdf")
-      end
-    end
+    @time = time { @users = User.all } # loads each attachment separately. don't know howto fix it.
+    # includes avatar_attachment doesn't make any difference. with_image_attachment either
```

Comment:
Here I try to track how attachments load

Reply:
Consider using sprite image to store all avatars